mantidproject/mantid
# SEQ new idf

Description of work.

**To test:**

Load data before and on Nov 13, 2017 and data after and on Nov 14 to check the difference. It is most obvious for short packs around the beam

Fixes #21184 

**Release Notes** 

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
